pkg/detector/library/driver.go

```diff
@@ -51,6 +51,9 @@ func NewDriver(libType string) (Driver, error) {
 	case ftypes.Pipenv, ftypes.Poetry, ftypes.Pip, ftypes.PythonPkg:
 		ecosystem = vulnerability.Pip
 		comparer = pep440.Comparer{}
+	case ftypes.Hex:
+		ecosystem = vulnerability.Erlang
+		comparer = compare.GenericComparer{}
```

Comment:
Which version spec Hex uses? [Semantic versioning](https://semver.org/) is widely used, but some languages use different spec like [PEP440](https://peps.python.org/pep-0440/).

Reply:
Hex uses semver 2.0 - https://hexdocs.pm/elixir/Version.html

Reply:
This document is about Elixir modules, isn't it? Do Hex packages also follow semver?

Reply:
i found info about publish new hex packages.
It says "First, make sure that the :version property is correct. All Hex packages are required to follow [semantic versioning](http://semver.org/)."
https://hex.pm/docs/publish#adding-metadata-to-code-classinlinemixexscode

Reply:
There is also `version module` information in the [hex documents](https://hex.pm/docs/usage#defining-dependencies). But the link to the elixir modules (which I posted earlier)

Reply:
Great! We can use the semver library.